WooCommerce/Classes/POS/Presentation/Card Present Payments/Connection Alerts/PointOfSaleCardPresentPaymentAlertType.swift

```diff
@@ -25,4 +25,12 @@ enum PointOfSaleCardPresentPaymentAlertType: Hashable, Identifiable {
     case connectingFailedUpdateAddress(viewModel: PointOfSaleCardPresentPaymentConnectingFailedUpdateAddressAlertViewModel)
     case connectingFailedUpdatePostalCode(viewModel: PointOfSaleCardPresentPaymentConnectingFailedUpdatePostalCodeAlertViewModel)
     case connectionSuccess(viewModel: PointOfSaleCardPresentPaymentConnectionSuccessAlertViewModel)
+
+    var isNonDismissable: Bool {
```

Comment:
I want to keep the modifier matching the convention that Apple have with `isInteractiveDismissDisabled` for sheets.

`isDismissDisabled` is fine with me though – "is non-dismissable" reads more as a fixed property of the object, rather than something that might change. In this iteration, it's fairly fixed, but it it's going to change more in future.

Reply:
8024ffa